generators/server/templates/gradle/profile_dev.gradle.ejs

```diff
@@ -45,15 +45,16 @@ bootRun {
 
 <%_ if (!skipClient) { _%>
 task webpackBuildDev(type: <%= _.upperFirst(clientPackageManager) %>Task) {
-    inputs.files(fileTree("<%= CLIENT_MAIN_SRC_DIR %>"))
+    inputs.file("package-lock.json")
+    inputs.dir("<%= CLIENT_MAIN_SRC_DIR %>")
 
     def webpackDevFiles = fileTree("<%= CLIENT_WEBPACK_DIR %>/")
     webpackDevFiles.exclude("webpack.prod.js")
     inputs.files(webpackDevFiles)
 
-    outputs.files(fileTree("<%= CLIENT_DIST_DIR %>"))
+    outputs.dir("<%= CLIENT_DIST_DIR %>")
 
-    dependsOn <%= clientPackageManager %>_install
+    dependsOn <%= clientPackageManager %>Install
```

Comment:
Did the forked plugin change the convention task names? I think we might update the documentation as well. I think there is also something in the generated review.

Reply:
No they are the same. The same tasks exist in the original plugin as well. The difference is that the ones with underscore are generated dynamically. See also [here](https://github.com/node-gradle/gradle-node-plugin/blob/master/docs/node.md#executing-npm-tasks)

Reply:
:heavy_check_mark: Does `yarnInstall` also exist? At least it is not explicitly mentioned in the documentation?

Reply:
Good question. I did not check but I assumed it does. Let me check in the code

Reply:
If I am not mistaken, it does. See https://github.com/node-gradle/gradle-node-plugin/blob/8ac7cbe248bae5c430cf4f9c6576db1e23b66501/src/main/groovy/com/moowork/gradle/node/NodePlugin.groovy#L57.

And a bit below that https://github.com/node-gradle/gradle-node-plugin/blob/8ac7cbe248bae5c430cf4f9c6576db1e23b66501/src/main/groovy/com/moowork/gradle/node/NodePlugin.groovy#L101  you can see the dynamic generation of the underscore variants, e.g: yarn_install